gradle.properties

```diff
@@ -11,4 +11,4 @@ mavenPassword=YourPassword
 # - here-naksha-lib-core/NakshaVersion (static property: latest)
 # - here-naksha-lib-psql/resources/naksha_plpgsql.sql (method: naksha_version)
 # - here-naksha-app-service/src/main/resources/swagger/openapi.yaml (info.version property)
```

Comment:
pls also change version to 2.1.2 here

File : here-naksha-app-service/src/main/resources/swagger/openapi.yaml
property : info.version